blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,79 @@ class EmberElectronBlueprint extends Blueprint {
   afterInstall(/* options */) {
     let logger = new Logger(this);
 
-    return this._installDependencies(logger)
-      .then(() => this._createResourcesDirectories())
-      .then(() => this._ensurePackageJsonConfiguration())
-      .then(() => this._updateGitignore())
```

Comment:
Okay, I guess we can fix it in another PR, but it seems weird to delete the code that does what we want to do in one PR just to re-add it in another PR...